submit/templates/submit/file_process.html

```diff
@@ -38,8 +38,10 @@
       <p>What to do next:</p>
       <ul>
         <li>review the log warnings and errors</li>
-        <li>correct any missing file problems</li>
+        <li>correct any missing file problems.
+        <li>make sure links to included files match the names of the files exactly (it is case sensitive)</li>
         <li>edit your files if needed</li>
+        <li>double check captions, references and citations.</li>
```

Comment:
I recommend putting references first"

```suggestion
        <li>verify your references, citations and captions.</li>
```